components/layout_2020/table/layout.rs

```diff
@@ -228,19 +240,21 @@ pub(crate) struct TableLayout<'a> {
 #[derive(Clone, Debug)]
 struct CellOrTrackMeasure {
     content_sizes: ContentSizes,
-    percentage: Percentage,
+    percentage: Option<Percentage>,
 }
 
 impl Zero for CellOrTrackMeasure {
     fn zero() -> Self {
         Self {
             content_sizes: ContentSizes::zero(),
-            percentage: Percentage(0.),
+            percentage: None,
         }
     }
 
     fn is_zero(&self) -> bool {
-        self.content_sizes.is_zero() && self.percentage.is_zero()
+        self.content_sizes.is_zero() &&
+            self.percentage
+                .is_none_or(|percentage| percentage.is_zero())
```

Comment:
It's maybe a bit strange that `measure.is_zero()` and `measure != CellOrTrackMeasure::zero()` can both be true.
Maybe this should just be 
```suggestion
                .is_none()
```